docs/devices/raspberry-pi-wpa2-enterprise.md

```diff
@@ -0,0 +1,143 @@
+# Raspberry Pi WPA2-Enterprise WiFi Setup
+
+This guide provides instructions for connecting Raspberry Pi devices to WPA2-Enterprise WiFi networks (commonly found in institutional environments). This setup is required for devices like the [picam](picam.md), [a1_cam](a1_cam.md), and other Raspberry Pi-based equipment in the AC Training Lab when operating on enterprise networks.
```

Comment:
Not sure if the markdown references are going to resolve, since there's some special documentation build steps that involve copying files. Don't change the way the documentation build happens, but do try to compile the docs and see if the links are valid